<launch>
	<arg
		name="o_device_name"
		default="mettler_optimax"/>
	<arg
		name="o_connection_mode"
		default="serial"/>
	<arg
		name="o_address"
		default="None"/>
	<arg
		name="o_port"
		default="/dev/ttyUSB0"/>
	<arg
		name="o_simulation"
		default="False"/>
	<node
		name="mettler_optimax"
		pkg="roslabware_drivers"
		type="mettler_optimax"
		args="
			-od $(arg o_device_name)
			-oc $(arg o_connection_mode)
			-oadd $(arg o_address)
			-op $(arg o_port)
			-ol $(arg o_simulation)"
		output="screen"/>
</launch>